Source/WebKit/UIProcess/Inspector/gtk/WebInspectorUIProxyGtk.cpp

```diff
@@ -81,6 +82,34 @@ static unsigned long long exceededDatabaseQuota(WKPageRef, WKFrameRef, WKSecurit
     return std::max<unsigned long long>(expectedUsage, currentDatabaseUsage * 1.25);
 }
 
+static void runOpenPanel(WKPageRef pageRef, WKFrameRef, WKOpenPanelParametersRef, WKOpenPanelResultListenerRef openPanelListener, const void *clientInfo)
+{
+    WTFLogAlways("RunOpenPanel\n");
```

Comment:
`WTFLogAlways` will print on the user's terminal, so it should be reserved for warnings and such, not debugging.

Reply:
Oops! I missed that from my debugging